hms-app/src/actions/GetBackendData.tsx

```diff
@@ -0,0 +1,52 @@
+const api_id = '6cyvp5ukhc'
+
+export const getListOfHackathons = (endpoint: any) => {
+  return fetch(
+    `${'http://localhost:4566/restapis/'}${api_id}${'/local/_user_request_'}/${endpoint}`,
+    {
+      method: 'GET',
+      headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json',
+      },
+    }
+  )
+    .then((response) => {
+      return response.json()
+    })
+    .catch((err) => console.log(err))
+}
+
+export const getHackathonDetails = (endpoint: any) => {
+  return fetch(
+    `${'http://localhost:4566/restapis/'}${api_id}${'/local/_user_request_/hackathon'}/${endpoint}`,
+    {
+      method: 'GET',
+      headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json',
+      },
+    }
+  )
+    .then((response) => {
+      return response.json()
+    })
+    .catch((err) => console.log(err))
+}
+
+export const getIdeaDetails = (endpoint: any) => {
```

Comment:
the argument could have a better name, something like `id` or `ideaId` so it is more obvious what should be passed in